source/ada/lsp-ada_handlers.adb

```diff
@@ -2114,26 +2115,147 @@ package body LSP.Ada_Handlers is
          is
             use Libadalang.Common;
 
-            Token : Token_Reference := First_Token (Node.Unit);
-            Name  : constant Wide_Wide_String :=
+            Token   : Token_Reference := First_Token (Node.Unit);
+            Name    : constant Wide_Wide_String :=
               Ada.Strings.Wide_Wide_Unbounded.To_Wide_Wide_String
                 (Get_Last_Name (Name_Node));
-            Span  : LSP.Messages.Span;
+            C       : LSP.Messages.TextEdit;
```

Comment:
Can you use more characters in names? I propose to replace `C` with `Text_Edit` for example.